### PR TITLE
Added normalize css.

### PR DESCRIPTION
**Pull request by mobywhale-chan**
- Renamed variables to use camelCase
- Moved some of normalize css into typography.scss. Basically anything that I think people will want to customise themselves.
- Moved list padding fix from normalize css to list skin (_listStandard.scss) - list that has bullet and numbers

### CODE REVIEW

File: components/_variables.scss [hunk 1]
```diff
@@ -9,30 +9,30 @@ $type-color: #333;
 
 
 /* typography */
-$base-line-height: 1.5em !default; //must be in em
-$base-font-size: 15px !default; // must be px
-$base-font-family: Georgia, "Times New Roman", Times, serif;
+$baseLineHeight: 1.5em !default; //must be in em
```
Comment (stubbornella) [inline components/_variables.scss:12]: Should this be used in the normalize file?

File: components/_variables.scss [hunk 2]
```diff
@@ -9,30 +9,30 @@ $type-color: #333;
...
+$baseFontSize: 15px !default; // must be px
```
Comment (stubbornella) [inline components/_variables.scss:13]: this might also be used, no?

**Comment by stubbornella** [comment]
I think we should discuss how we use variables within normalize to make a customized normalize per project, but I'll merge it and we can iterate.

**Comment by mobywhale-chan** [comment]
That sounds great. We can discuss after standup.

**Pull request merged**